### PR TITLE
CSL-21: PCA - Responsible officer details

### PR DESCRIPTION
**Pull request by dk4g**
## What? 
[CSL-21](https://collaboration.homeoffice.gov.uk/jira/browse/CSL-21) - 1.3 PCA - Responsible officer details (About the applicants)
Added screens to gather information about Responsible officer and current DBS certificate. 
## Anything Else? (optional)
Steps have been split into relevant sections and commented out to shorten the user journey to the summary page and speed up the testing process. 

Includes Bugfix [CSL-52](https://collaboration.homeoffice.gov.uk/jira/browse/CSL-52) - Optional field throws error with and without data in the field

## Check list

- [ ] I have reviewed my own pull request
- [ ] I have written tests (if relevant)

### CODE REVIEW

File: apps/precursor-chemicals/translations/src/en/fields.json [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+  },
+  "responsible-officer-dbs-application-fullname": {
+    "label": "Full name",
+    "hint": "Enter the name as it appears on their DBS application"
+  },
+  "responsible-officer-dbs-reference": {
+    "label": "DBS reference"
+  },
+  "responsible-officer-dbs-date-of-issue": {
+    "label": "Date of issue or application"
```
Comment (vinodhasamiyappanHO) [inline apps/precursor-chemicals/translations/src/en/fields.json:19]: change this `label` to `legend`

File: apps/precursor-chemicals/fields/index.js [hunk 2]
```diff
@@ -1,2 +1,56 @@
...
+  'responsible-officer-dbs-certificate': {
+    mixin: 'checkbox',
+    validate: [ 'required' ]
+  },
+  'responsible-officer-dbs-application-fullname': {
+    mixin: 'input-text',
+    validate: [ 'required' ],
+    className: ['govuk-input', 'govuk-!-width-two-thirds']
+  },
+  'responsible-officer-dbs-reference': {
```
Comment (vinodhasamiyappanHO) [inline apps/precursor-chemicals/fields/index.js:23]: className: ['govuk-input', 'govuk-!-width-two-thirds'] - has to be added for this field to align with sigma

File: apps/precursor-chemicals/translations/src/en/pages.json
Comment (RobertMcCann) [inline apps/precursor-chemicals/translations/src/en/pages.json:1]: Update header on /responsible-officer-dbs-information to match figma:  'DBS information for the responsible officer'

**Comment by RobertMcCann** [comment]
I think we need notUrl validations on free input fields like the names, but I don't see any validation messages for that in the document - is this something we need to confirm with UCD ?

**Review by vinodhasamiyappanHO — APPROVED** [review]
Looks good to me

**Review by vinodhasamiyappanHO — APPROVED** [review]
Looks good to me

**Pull request merged**